scripts/build_wheel.py

```diff
@@ -0,0 +1,41 @@
+import os
+import logging
+import subprocess
+
+_logger = logging.getLogger(__file__)
+logging.basicConfig(level=logging.INFO)
+
+def build_client():
+    # Find our initial directory
+    _logger.info("Starting build_client")
+    _logger.info("Running npm install")
+    subprocess.check_call(
+        ['npm', 'install'],
+        cwd='client',
+        shell=True
+    )
```

Comment:
Might just be user error on my side, but I run into this error when I hit this code running the script from my adatest base repository:

```bash
npm@8.3.1 /home/harsha/.nvm/versions/node/v16.14.0/lib/node_modules/npm
Traceback (most recent call last):
  File "scripts/build_wheel.py", line 41, in <module>
    main()
  File "scripts/build_wheel.py", line 33, in main
    build_client()
  File "scripts/build_wheel.py", line 12, in build_client
    subprocess.check_call(
  File "/home/harsha/miniconda3/envs/adatest-dev/lib/python3.8/subprocess.py", line 364, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
```

`cd client && npm install` works. Am I doing something wrong?

Reply:
The `cwd` argument to `check_call()` is supposed to do that :-/

Reply:
Would be helpful if `subprocess` printed out the actual error, though.

I take it that just running `npm install` yourself worked fine?

Reply:
Yeah, running `cd client && npm install` works fine from root

Reply:
Hmmm.... just reread the `subprocess` docs a little more closely, and have pushed a change. Does that give a more informative error?